pkg/controller/reconciler/users_reconciler.go

```diff
@@ -0,0 +1,65 @@
+package reconciler
+
+import (
+	"fmt"
+
+	"github.com/oslokommune/okctl/pkg/apis/okctl.io/v1alpha1"
+
+	"github.com/oslokommune/okctl/pkg/client"
+	"github.com/oslokommune/okctl/pkg/controller/resourcetree"
+)
+
+// UsersState contains runtime data needed in Reconcile()
+type UsersState struct {
+	UserPoolID string
+	Users      []v1alpha1.ClusterUser
```

Comment:
This seems unused

Reply:
It is, I started on the delete side of the operations, and then saw the functionality was missing, and didn't think it was that important at this stage 🙈

Reply:
Ah good point